src/containers/User/UserArea.tsx

```diff
@@ -460,7 +460,7 @@ const UserMenu: React.FC<{ user: User; templates: TemplateInList[] }> = ({ user,
       <Dropdown.Item
         icon="log out"
         text={t('MENU_LOGOUT')}
-        onClick={() => showLogoutModal({ onConfirm: logout })}
+        onClick={() => (isMobile ? showLogoutModal({ onConfirm: logout }) : logout())}
```

Comment:
Don't think we need logout confirmation on Mobile, it hasn't actually been an issue. Handy for the mobile menu though as it's much easier to accidentally tap the wrong item.

Reply:
Agreed.